licenses/inlong-sort-connectors/LICENSE

```diff
@@ -874,6 +874,7 @@ The text of each license is the standard Apache 2.0 license.
   com.cedarsoftware:json-io:2.5.1 - Java JSON serialization (https://github.com/jdereg/json-io), (Apache License, Version 2.0)
   net.minidev:json-smart:2.3 - JSON Small and Fast Parser (https://github.com/netplex/json-smart-v2/tree/v2.3), (The Apache Software License, Version 2.0)
   com.github.jsqlparser:jsqlparser:2.1 - JSQLParser library (https://github.com/JSQLParser/JSqlParser), (The Apache Software License, Version 2.0;  GNU Library or Lesser General Public License (LGPL) V2.1)
```

Comment:
Do we need to remove the old version from the license?

Reply:
removed